### PR TITLE
Fixes #28460 - adding eslint-plugin-foreman for styling

### PR DESCRIPTION
**Pull request by amirfefer**
blocked by https://github.com/theforeman/foreman-js/pull/87

### CODE REVIEW

**Comment by amirfefer** [comment]
Screenshot from 2019-12-11 19-13-41

**Comment by amirfefer** [comment]
keep in mind that `@theforeman v3.7.0` is broken until https://github.com/theforeman/foreman-js/pull/92 will be merge

**Comment by amirfefer** [comment]
tests failures related to the new version of redux

**Comment by MariaAga** [comment]
Test failures will be fixed in jest 25 
https://github.com/facebook/jest/issues/9216

**Comment by adamruzicka** [comment]
Needs a rebase

File: webpack/ForemanTasks/Components/TasksDashboard/Components/TasksTimeRow/TasksTimeRow.stories.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```
Comment (amirfefer) [inline webpack/ForemanTasks/Components/TasksDashboard/Components/TasksTimeRow/TasksTimeRow.stories.js]: @sharvit - using hooks outside of a component gives an error, transforming this to `WithState` solve it

**Comment by amirfefer** [comment]
@adamruzicka  - rebased

**Pull request merged**